packages/wp-story-editor/src/components/documentPane/status/status.js

```diff
@@ -37,6 +48,37 @@ import {
  */
 import { VISIBILITY, STATUS } from '../../../constants';
 
+const StyledListItem = styled.li`
+  position: relative;
+  padding: 6px 8px;
+  margin: 4px 8px;
+  align-items: center;
+  border-radius: 4px;
+  cursor: pointer;
+  ${themeHelpers.focusableOutlineCSS}
+  &:hover {
+    background-color: ${theme.colors.border.defaultNormal};
+  }
+`;
+const LabelText = styled.span`
+  font-size: 14px;
+  color: ${theme.colors.fg.primary};
+`;
+
+const HelperText = styled.span`
+  color: ${theme.colors.fg.tertiary};
+`;
```

Comment:
💭 Should these be styled `Text` elements? May not be worth the trouble since this is so easy